analytics/filesystem/file_module.go

```diff
@@ -98,6 +108,23 @@ func jsonifyAuctionObject(ao *analytics.AuctionObject) string {
 	}
 }
 
+func jsonifyVideoObject(vo *analytics.VideoObject) string {
+	type alias analytics.VideoObject
+	b, err := json.Marshal(&struct {
+		Type RequestType `json:"type"`
+		*alias
+	}{
+		Type:  VIDEO,
+		alias: (*alias)(vo),
+	})
+
+	if err == nil {
+		return string(b)
+	} else {
+		return fmt.Sprintf("Transactional Logs Error: Video object badly formed %v", err)
+	}
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/15863840/70836646-dc9d8280-1dce-11ea-80ab-b25ba066c103.png)
there seems to not be test cases for `jsonifyVideoObject` either. What if the parameter is `nil`? Are we ok with a JSON object with a `nil` field? Do we need to check for `nil`? I realized this is a local function but if we want to extend in the future, corner cases wouldn't hurt.

Reply:
Yes good point Gus, I added some calls to test the video endpoint logging and jsonify also in the new commit. Regarding checking for nil fields in jsonifyVideoObject I think it will be ok because if a field is nil it will show up in the JSON with a null value, which the person evaluating the log file should intuitively associate with a nil value in Go. Thanks for the feedback.